deploy/clusterrole.yaml

```diff
@@ -0,0 +1,29 @@
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: acme-controller
+  annotations:
+    openshift.io/reconcile-protect: "true"
+rules:
+- apiGroups:
+  - ""
```

Comment:
This is the default value that is returned by the API when you get some of the system roles. I can set it to null if you prefer.